src/components/QuranReader/TranslationView/TranslationViewCell.module.scss

```diff
@@ -15,6 +15,14 @@
   flex: 1;
   padding-top: var(--spacing-medium);
   position: relative;
+  @include breakpoints.tablet {
```

Comment:
1. let's do it only for 1 verse. 
2. Adding top padding that's dependent on the quranic font size